core/server/common/src/main/java/alluxio/master/BackupManager.java

```diff
@@ -332,6 +336,7 @@ public void initFromBackup(InputStream is) throws IOException {
         mRestoreTimeMs = System.currentTimeMillis() - startRestoreTime;
         mRestoreEntriesCount = appliedEntryCount.get();
         traceExecutor.shutdownNow();
+        es.shutdown();
```

Comment:
i might need to make this part much more uglier to make sure the 2nd shutdown is called even if the 1st one failed